demo/client/src/view.ts

```diff
@@ -197,6 +202,60 @@ export abstract class DemoView extends PathfinderView implements RenderContext {
 
         this.atlasRenderingTimerQuery = null;
         this.compositingTimerQuery = null;
+
+        this.inVRRAF = false;
+        this.inVR = false;
+        this.vrDisplay = null;
+        if ("VRFrameData" in window) {
+           this.vrFrameData = new VRFrameData;
+        } else {
+            this.vrFrameData = null;
+        }
+
+        this.vrSetup();
+    }
+
+    vrSetup(): void {
+        if (navigator.getVRDisplays) {
+            navigator.getVRDisplays().then((displays) => {
+              if (displays.length > 0) {
+                this.vrDisplay = displays[displays.length - 1];
+
+                // It's heighly reccommended that you set the near and far planes to
+                // something appropriate for your scene so the projection matricies
```

Comment:
typo: should be "matrices"

Reply:
lol this is a comment from the demo app, I should fix that